packages/docs/docs/medications/formulary.md

```diff
@@ -0,0 +1,298 @@
+---
+keywords:
+  - drugs
+  - formulary
+  - medication
+---
+
+# Modeling a Formulary
+
+## Introduction
+
+A formulary is a catalog of drugs offered by your organization. When implementing a custom EMR, you may want to include essential metadata about these drugs to help prescribing physicians, pharmacists, and patients.
+
+The principal resource for building a digital formulary is the [`MedicationKnowledge`](/docs/api/fhir/resources/medicationknowledge) resource. This guide aims to explore the most pertinent attributes of [`MedicationKnowledge`](/docs/api/fhir/resources/medicationknowledge) for digital health providers.
```

Comment:
Consider one more sentence:  In a formulary implementation MedicationKnowledge represents a type of type of medication that can be ordered.